src/pybind/mgr/influx/module.py

```diff
@@ -108,7 +109,46 @@ def get_df_stats(self):
                     }
                 }
                 data.append(point)
-        return data
+                pool_info.update({str(pool['id']):pool['name']})
+        return data, pool_info
+
+    def get_pg_summary(self, pool_info):
+        osd_sum = self.get('pg_summary')['by_osd']
+        pool_sum = self.get('pg_summary')['by_pool']
+        data = []
+        for osd_id, stats in osd_sum.iteritems():
+            metadata = self.get_metadata('osd', "%s" % osd_id)
+            for stat in stats:
+                point_1 = {
+                    "measurement": "ceph_pg_summary_osd",
+                    "tags": {
+                        "ceph_daemon": "osd." + str(osd_id),
+                        "type_instance": stat,
+                        "host": metadata['hostname']
+                    },
+                    "time" : datetime.utcnow().isoformat() + 'Z', 
```

Comment:
Suggest you format the time value once at the start of the function rather than for each value